syscalls_windows.go

```diff
@@ -0,0 +1,171 @@
+// +build windows
+
+// To use it with windows, you need a tap driver installed on windows.
+// https://github.com/OpenVPN/tap-windows6
+// or just install OpenVPN
+// https://github.com/OpenVPN/openvpn
+package water
+
+import (
+	"errors"
+	"net"
+	"os"
+	"syscall"
+
+	"golang.org/x/sys/windows/registry"
+)
+
+var (
```

Comment:
Go doesn't have `const` functions, so using `const` means you can't reuse these bit-wise code through [`tap_control_code`](https://github.com/songgao/water/pull/13/files#diff-6be27a31703495803c6129f15a873036R40) below.

All those `tap_*` are only used within the `water` package, so I think it's fine to leave this as `var`.